quantus/metrics/robustness/relative_representation_stability.py

```diff
@@ -363,9 +333,7 @@ def evaluate_batch(
             )
 
             # Generate explanations for perturbed input.
-            a_batch_perturbed = self.generate_normalised_explanations_batch(
```

Comment:
nice to remove this one, have you deleted that function now?

Reply:
Yes, I removed it because `explain_batch` does the same.